tests/e2e/specs/out-of-game/stats.spec.js

```diff
@@ -45,10 +45,12 @@ function setup() {
       const matches = matchesFixture.map(transformMatchFixture);
 
       const games = gameFixtures.map((game) => {
+        const winner = game.winner === game.p0 ? this[game.p0] : this[game.p1];
```

Comment:
What about games that stalemate? I think we need to allow for the possibility that `game.winner` is undefined, which is how we'd represent a stalemate. I can't think of a super clean way to accommodate both the null check and the comparison to p0 vs p1, but maybe we can just use a conditional
```suggestion
        let winner = null;
        if (game.winner) {
          winner = game.winner === game.p0 ? this[game.p0] : this[game.p1];
        }
```